addons/pos_self_order/static/src/LandingPageComponents/LandingPageHeader/LandingPageHeader.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="LandingPageHeader" owl="1">
+        <div class="m-3">
+            <img src="/logo?company=#{pos.company.id}" alt="logo" class="rounded"/>
```

Comment:
I don't see any declaration of "pos" in the `LandingPageHeader` component. Also, do you mean `t-att-src`?